src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -553,6 +553,8 @@ def _compute_offsets(self, token_ids, time_precision=0.02):
                 The time ratio to convert from token to time.
         """
         offsets = []
+        if hasattr(token_ids, "cpu") and callable(token_ids.cpu):
```

Comment:
We don't want to import PyTorch in the tokenizer file, so instead we check whether the token ids have the method `.cpu()` defined, and execute this if so

Is this ok with you @ArthurZucker?

Reply:
would probably have checked the type `if "torch" in str(type(token_ids)):` which might be less brittle? but alright otherwise